### ISSUE TITLE
Update to MySqlConnector 0.31.0

### ISSUE THREAD

**Issue by jemiller0** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#440]
Can you release a new provider that makes use of MySqlConnector 0.31.0? Something appears to have changed as of version 0.30.0 of MySqlConnector. I saw something in the git log about how it's using a strong name now. So, it looks like the publicKeyToken went from being null to d33d3e53aa5f8c92. I tried to create a binding redirect like the following, but, have been unable to get it to work. I was using version 0.29.4 before. For some reason, I no longer see 0.29.4 in NuGet and only see 0.26.4. So, I am unable to revert to the version I was using before and have to use an older version.

```
      <dependentAssembly>
        <!--<assemblyIdentity name="MySqlConnector" publicKeyToken="null" culture="neutral" />-->
        <assemblyIdentity name="MySqlConnector" publicKeyToken="d33d3e53aa5f8c92" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-0.31.0.0" newVersion="0.31.0.0" />
      </dependentAssembly>

```

**Comment by yukozh** [comment]
You can just install the MySqlConnector 0.31.0 via nuget gallery in VS, then it will override the old one.

**Comment by jemiller0** [comment]
That's what I'm telling you. It doesn't work. Because MySqlConnector changed it's publicKeyToken. Try it and you will see.

**Comment by mguinness** [comment]
This appears to be related to https://github.com/mysql-net/MySqlConnector/issues/224.

**Comment by caleblloyd** [comment]
Looks like @yukozh just published [2.0.1-preview-10078 on myget](https://www.myget.org/feed/pomelo/package/nuget/Pomelo.EntityFrameworkCore.MySql/2.0.1-preview-10078) with MySqlConnector 0.31
